### PR TITLE
clarify some old tips are not relevant today

### PR DESCRIPTION
**Pull request by A-viral-dev**
Last two tips on https://onextrapixel.com/10-principles-for-keeping-your-programming-code-clean/

 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [x] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
- [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1.Describe the changes made and include why they are necessary or important:

...your text here

#### 2. Related Issue

keyword #XXXXX

### CODE REVIEW

**Comment by twalton83** [comment]
How are they no longer relevant?

**Comment by A-viral-dev** [comment]
1. The First tip says that you should avoid intermixing code languages, but JS framework like React and svelte does precisely that. 
2. Multiple CSS stylesheet import is no longer a constraint in performance. Instead of one StyleSheet, dividing your CSS into components is a much-preferred style nowadays. It leads better readability and faster load time. Modern browsers add files to their cache, and that includes those CSS files. They can also do parallel downloads, meaning that all files are downloaded simultaneously by using multiple threads or other options. (Asynchronous requests.) The way modern browsers will optimize file handling can impact the speed of one large CSS file versus several small ones.

**Comment by leosoaivan** [comment]
@A-viral-dev 

I think point 9 refers to instances were a developer might literally inline CSS and script tags within HTML, which, while valid, could end up being messy. I do not think it refers to intermixing of CSS, HTML, and JS you might find in JSX. In a vanilla setup, it'd make sense to have your stylesheets and script tags separated. So in my view, the point still stands, but is moot.

I agree with your stance on point 10.

This LGTM, but could we also consider linking to other resources? I'd be interested in alternatives, including tips that are applicable to broader aspects of programming, not necessarily focused on web development. Care to take a look around @A-viral-dev?

**Comment by twalton83** [comment]
@A-viral-dev thoughts on the above?

File: foundations/javascript_basics/clean_code.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ There are many different opinions on what constitutes great JavaScript code.  Th
 
 Read through these articles that discuss a few elements of writing good clean code.
 
-1. [This list of clean-code tips](https://onextrapixel.com/10-principles-for-keeping-your-programming-code-clean/).
+1. [This list of clean-code tips](https://onextrapixel.com/10-principles-for-keeping-your-programming-code-clean/). You can ignore last two tips as they are no longer relevant in current programming environment.
```
Comment (twalton83) [inline foundations/javascript_basics/clean_code.md:90]:
```suggestion
1. [This list of clean-code tips](https://onextrapixel.com/10-principles-for-keeping-your-programming-code-clean/). You can ignore last two tips as they are no longer relevant in the current programming environment.
```

**Review by twalton83 — COMMENTED** [review]
Small grammar issue.

**Comment by twalton83** [comment]
@A-viral-dev Interested in addressing the above? Otherwise I'll close this.

**Comment by thatblindgeye** [comment]
Closing this due to inactivity/lack of response after almost 2 months. @A-viral-dev if you would still like this change to be committed, please review the comments left above and open a new PR.

**Pull request closed without merging**